v3/plugins/eclipse/che-machine-exec-plugin/nightly/meta.yaml

```diff
@@ -27,3 +27,5 @@ spec:
      image: "quay.io/eclipse/che-machine-exec:nightly"
      ports:
        - exposedPort: 4444
+     command: ['/go/bin/che-machine-exec', '--static', '/cloud-shell', '--url', '127.0.0.1:4444']
```

Comment:
@metlos Does it influence Single-user Che where we don't run jwtproxy?

Reply:
Seems it breaks Single-User Che https://github.com/eclipse/che/issues/16053

Reply:
I think @metlos in PTO.